nixos/modules/programs/wayland/sway.nix

```diff
@@ -123,12 +123,23 @@ in
             # Import the most important environment variables into the D-Bus and systemd
             # user environments (e.g. required for screen sharing and Pinentry prompts):
             exec dbus-update-activation-environment --systemd DISPLAY WAYLAND_DISPLAY SWAYSOCK XDG_CURRENT_DESKTOP
```

Comment:
this line passes `--systemd`, described as "Also update the 'systemd --user' environment if possible."

So I'm wondering, is the `systemctl --user import-environment` line necessary?

And, a bit off-topic, shouldn't the `exec dbus-update-activation-environment ...` line be an `exec_always`?

(When I rebuild my home-manager config, certain services I have that need WAYLAND_DISPLAY seem to be unable to restart, and I've found--after noticing this PR, thanks!--that manually running `systemctl --user import-environment DISPLAY WAYLAND_DISPLAY SWAYSOCK;` then enables the services to restart successfully.)

Reply:
> (When I rebuild my home-manager config, certain services I have that need WAYLAND_DISPLAY seem to be unable to restart, and I've found--after noticing this PR, thanks!--that manually running systemctl --user import-environment DISPLAY WAYLAND_DISPLAY SWAYSOCK; then enables the services to restart successfully.)

Doesn't this indicate that it's still necessary to run `systemctl --user import-environment`? It says "if possible" but who knows what "possible" means.

Reply:
Interesting point on the "if possible." OK, I agree, it sounds safer to leave it in!